lumicks/pylake/kymo.py

```diff
@@ -28,6 +28,22 @@ def _default_line_time_factory(self: "Kymo"):
         )
 
 
+def _default_line_timestamp_ranges_factory(self: "Kymo", exclude: "bool"):
```

Comment:
```suggestion
def _default_line_timestamp_ranges_factory(self: "Kymo", exclude: bool):
```
No need for the quotes on bool.